composer.json

```diff
@@ -16,30 +16,31 @@
         "mpdf/mpdf": "^v8.1.2",
         "phpoffice/phpspreadsheet": "^1.25",
         "stof/doctrine-extensions-bundle": "^1.7",
-        "symfony/console": "6.2.*",
-        "symfony/doctrine-messenger": "6.2.*",
-        "symfony/dotenv": "6.2.*",
+        "symfony/console": "~6.3.0",
+        "symfony/doctrine-messenger": "~6.3.0",
+        "symfony/dotenv": "~6.3.0",
         "symfony/flex": "^2",
-        "symfony/form": "6.2.*",
-        "symfony/framework-bundle": "6.2.*",
-        "symfony/http-client": "6.2.*",
-        "symfony/messenger": "6.2.*",
+        "symfony/form": "~6.3.0",
+        "symfony/framework-bundle": "~6.3.0",
+        "symfony/http-client": "~6.3.0",
+        "symfony/messenger": "~6.3.0",
         "symfony/monolog-bundle": "^3.8",
-        "symfony/proxy-manager-bridge": "6.2.*",
-        "symfony/runtime": "6.2.*",
-        "symfony/security-bundle": "6.2.*",
-        "symfony/translation": "6.2.*",
-        "symfony/twig-bundle": "6.2.*",
-        "symfony/validator": "6.2.*",
-        "symfony/webpack-encore-bundle": "^1.15",
-        "symfony/yaml": "6.2.*"
+        "symfony/proxy-manager-bridge": "~6.3.0",
+        "symfony/runtime": "~6.3.0",
+        "symfony/security-bundle": "~6.3.0",
+        "symfony/stimulus-bundle": "^2.10",
+        "symfony/translation": "~6.3.0",
+        "symfony/twig-bundle": "~6.3.0",
+        "symfony/validator": "~6.3.0",
+        "symfony/webpack-encore-bundle": "^2.0",
+        "symfony/yaml": "~6.3.0"
```

Comment:
The `symfony` versions are customarily specified as `6.3.*`, but since “Tilde tælopper terminale tal” `~6.3.0` is just the same.

Reply:
This is ergebnis/composer-normalize that enforces this change.